imnodes.cpp

```diff
@@ -977,16 +994,35 @@ void click_interaction_update(EditorContext& editor)
             g.style.link_thickness,
             link_data.num_segments);
 
-        if (left_mouse_released)
+        const bool link_creation_on_snap = 
+            (editor.pins.pool[g.hovered_pin_idx.value()].flags & AttributeFlags_EnableLinkCreationOnSnap);
```

Comment:
This will cause the assert inside `value()` to trigger when we're not actually hovering near a pin, so we should conditionally test for `AttributeFlags_EnableLinkCreationOnSnap`, or get `false`.

Reply:
Fixed